doc/rtd/explanation/failure_states.rst

```diff
@@ -0,0 +1,126 @@
+.. _failure_states:
+
+Failure states
+==============
+
```

Comment:
> It's not ideal to have sequential headers with no text in between them

Agreed, fixed.

> Could we take a second look at the header structure to avoid that?

I've broken some of this content out into two new pages:
```
doc/rtd/explanation/exported_errors.rst
doc/rtd/howto/status.rst
```